src/modules/App/stories/utils.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 
-export const fitPageSize = (component) => (
+export const fitPageSize = (component, style = {}) => (
```

Comment:
do we use styles somewhere and is not applied? curious

Reply:
no, I added this just because it's useful when we test it in local